docs/INSTALLATION.md

```diff
@@ -0,0 +1,89 @@
+## Prerequisites
+
+The Fortran language track requires that you have the following software
+installed on your system:
+* a modern Fortran compiler
+* the CMake cross-platform build system
+
+### Prerequisite: A Modern Fortran Compiler 
+
+This language track requires a compiler with [Fortran 2003](https://en.wikipedia.org/wiki/Fortran#Fortran_2003)
+support. All major compilers released in the last few years should
+be compatible, so as long as you are on a fairly recent version you should be fine.
+
+The following will describe installation of [GNU Fortran](https://gcc.gnu.org/fortran/) or GFortran. Other fortran compilers are listed [here](https://en.wikipedia.org/wiki/List_of_compilers#Fortran_compilers). Intel Fortran is a popular proprietary choice for high performance applications. Most exercises will work with Intel Fortran, but are only tested with GNU Fortran so your milage may vary. 
```

Comment:
```suggestion
The following describes the installation of [GNU Fortran](https://gcc.gnu.org/fortran/) or GFortran.
Other fortran compilers are listed [here]https://en.wikipedia.org/wiki/List_of_compilers#Fortran_compilers).
Intel Fortran is a popular proprietary choice for high performance applications.
Most exercises will work with Intel Fortran, but are only tested with GNU Fortran so your milage may vary. 
```

If we do one sentence per line, the corrections (diffs) won't be as noisy, we hope.  Up to you, though how we do this, as "Chief Maintainer".  I am not opposed to 80 character (or thereabout) line limit, but line per sentence has its benefits over 80 characters per line.